site/content/api/Tenant-API.md

```diff
@@ -242,7 +245,7 @@ The table below provides an overview of the standard members defined for the JSO
 | Name                     | Mandatory | Type       | Default Value | Description |
 | :------------------------| :-------: | :--------- | :------------ | :---------- |
 | *tenant-id*              | *yes*     | *string*   |               | The ID of the tenant. |
-| *enabled*                | *no*      | *boolean*  | `true`        | If set to `false` the tenant is currently disabled.|
+| *enabled*                | *no*      | *boolean*  | `true`        | If set to `false` the tenant is currently disabled. Devices of disabled tenants can no longer interact with any *Hono protocol adapter*. *Clients* of disabled tenants can no longer interact with the *Hono AMQP 1.0 Messaging Network*|
```

Comment:
Since the *messaging network* is not *per se* a part of Hono, there is no way to tell if consumers can or cannot interact with the messaging network if a tenant is disabled. We should specify the impact from a Hono  standpoint only. We can, for example, say that protocol adapters MUST NOT allow devices of a disabled tenant to connect and MUST NOT accept data published by such devices. This would also imply that downstream consumers would not be able to receive any data originating from such devices.

Reply:
You're right. I adapted the description accordingly.